src/main/scala/com/suprnation/actor/test/TestKit.scala

```diff
@@ -51,9 +51,8 @@ trait TestKit {
     for {
       startQ <- actor.messageBuffer
       _ <- Console[F].println(s"Expecting: $messages in queue: $startQ")
-      expectedQ = startQ._2 ++ messages
       _ <- awaitCond(
-        actor.messageBuffer.map(_._2 == expectedQ),
+        actor.messageBuffer.map(_._2.intersect(messages) == messages),
```

Comment:
In my test cases, `startQ` would inconsistently either be empty or contain the first message. Comparing the intersection with the expected is more robust because it removes the race condition from getting the initial buffer